manifests/k/KhronosGroup/VulkanSDK/1.3.268.0/KhronosGroup.VulkanSDK.installer.yaml

```diff
@@ -0,0 +1,26 @@
+# Created using wingetcreate 1.5.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: KhronosGroup.VulkanSDK
+PackageVersion: 1.3.268.0
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: exe
+Scope: machine
+InstallModes:
+- interactive
+- silent
+InstallerSwitches:
+  Silent: --accept-licenses --default-answer --confirm-command install
+  SilentWithProgress: --accept-licenses --default-answer --confirm-command install
+UpgradeBehavior: install
+Dependencies:
+  PackageDependencies:
+  - PackageIdentifier: Microsoft.VCRedist.2015+.x64
```

Comment:
The error @stephengillie was seeing starting `vkcube.exe` was due to the fact that the Vulkan Runtime was not installed. This is normally taken care of by your graphics card drivers but you can also update this manually. I've created a new PR for adding VulkanRT too #124997 and we could add that as a dependency for the SDK maybe?